lib/licensed/sources/nuget.rb

```diff
@@ -188,16 +189,27 @@ def enumerate_dependencies
             # Already know this package from another target
             next if dependencies.key?(id)
 
-            path = File.join(nuget_packages_dir, json["libraries"][reference_key]["path"])
-            dependencies[id] = NuGetDependency.new(
-              name: id,
-              version: version,
-              path: path,
-              metadata: {
-                "type" => NuGet.type,
-                "name" => name
-              }
-            )
+            # Find package in one of nuget folders
+            nuget_packages_dirs.each do |packages_dir|
+              path = File.join(packages_dir, json["libraries"][reference_key]["path"])
+              if (File.directory?(path))
+                dependencies[id] = NuGetDependency.new(
+                  name: id,
+                  version: version,
+                  path: path,
+                  metadata: {
+                    "type" => NuGet.type,
+                    "name" => name
+                  }
+                )
+                break
+              end
+            end
+
+            # Throw if package not found
+            unless dependencies.key?(id)
+              raise "Package #{id} not found, looked in #{nuget_packages_dirs}"
```

Comment:
@jonabc to confirm that it's acceptable to raise here while enumerating dependencies (as opposed to when determining license for a dependency)

Reply:
yeah this shouldn't raise an error or it will halt dependency enumeration.  the app generally handles cases when the path doesn't exist and will raise a warning as appropriate.  the fix I'd suggest here is to gracefully handle when the `NuGetDependency.nuspec_path` file doesn't exist.